python/packages/isce3/atmosphere/ionosphere_filter.py

```diff
@@ -952,13 +955,13 @@ def remove_small_components(image, min_cluster_pixels):
     n_valid = int(valid_mask.sum())
 
     if min_cluster_pixels == 0:
-        print("[remove_small_components] min_cluster_pixels == 0 → "
-              "no components will be removed.")
+        info_channel.log("[remove_small_components] min_cluster_pixels == 0 → "
+                         "no components will be removed.")
         return image.copy()
 
     if min_cluster_pixels >= n_valid:
-        raise ValueError(
-            f"min_cluster_pixels ({min_cluster_pixels}) must be smaller than "
+        info_channel.log(
+            f"min_cluster_pixels ({min_cluster_pixels}) are lager than "
             f"the total number of valid pixels ({n_valid})."
         )
```

Comment:
```suggestion
        info_channel.log(
            f"min_cluster_pixels ({min_cluster_pixels}) are larger than "
            f"the total number of valid pixels ({n_valid})."
        )
```

Typo. Also here, it is more `appropriate` to use a `warning` channel.

Reply:
@vbrancat I agree with you. `warning` is better than `info` in this sentence. and thank you for picking the typo here :)